### DISCUSSION TITLE
bS Cookie Consent + WPML

### DISCUSSION THREAD

**Discussion by balrogmedia** [bootscore/bootscore#296] [Q&A]
Hello,
I am using bS Cookie Consent and WPML as my site is multi language.

The link to the privacy policy page on the cookie consent bar is fine for EN, but it adds a slash "/" too much for the other languages, for example

www.sitename.com/privacy-policy
www.sitename.com/it//privacy-policy
www.sitename.com/no//privacy-policy

is this a recurring issue? any idea how to fix it please?

thanks
![Screenshot 2022-10-19 at 13 14 20](https://user-images.githubusercontent.com/98542232/196676250-144bdfa0-2119-4e01-8bff-bef393568b1d.png)

**Comment by crftwrk** [comment]
You have to init Cookie Consent for each language and change link there. Copy each init code to an own widget. Now you can manage with WPML on which language it should be shown.

Here done with Polylang, but WPML is this nearly the same:

<img width="347" alt="screenshot" src="https://user-images.githubusercontent.com/51531217/196680505-666f58cf-59de-4ecf-98ac-c88fcaa72c93.png">

**Reply by balrogmedia** [reply]
Thank you but I'm a bit confused... how do I copy a plugin in a widget?

**Reply by crftwrk** [reply]
As described here https://bootscore.me/documentation/plugin/bs-cookie-settings/#Init, the plugin installation itself does nothing. You must initialize it with JavaScript. You should add this init snippet in a custom html widget in footer 4 position. Now you can handle widgets with WPML.

**Reply by crftwrk** [reply]
Ah sorry, my bad. I see you are using the old cookie plugin  and solution above is wrong. You may check the WPML settings for this or do redirects.

But this plugin is not ready for the european law anymore. That‘s why we created a new and better one. Check this here https://github.com/bootscore/bs-cookie-settings
